EngineLayer/ProteinParsimony/ProteinGroup.cs

```diff
@@ -349,11 +358,13 @@ public void CalculateSequenceCoverage()
         {
             var proteinsWithUnambigSeqPsms = new Dictionary<Protein, List<PeptideWithSetModifications>>();
             var proteinsWithPsmsWithLocalizedMods = new Dictionary<Protein, List<PeptideWithSetModifications>>();
+            var proteinsWithUnambigSeqPsmsPSM = new Dictionary<Protein, HashSet<PeptideSpectralMatch>>();
 
             foreach (var protein in Proteins)
             {
                 proteinsWithUnambigSeqPsms.Add(protein, new List<PeptideWithSetModifications>());
                 proteinsWithPsmsWithLocalizedMods.Add(protein, new List<PeptideWithSetModifications>());
+                proteinsWithUnambigSeqPsmsPSM.Add(protein, new HashSet<PeptideSpectralMatch>());
             }
 
             foreach (var psm in AllPsmsBelowOnePercentFDR)
```

Comment:
let's make sure that this is the field from ProteinGroup and not the complete set of psms from the search

variable should be protein.AllPsmsBelowOnePercentFDR

Reply:
Confirmed that we're looping through the psms that correspond to the protein group. AllPsmsBelowOnePercentFDR